### ISSUE TITLE
How to migrate consumers form eager to cooperative rebalance? 

### ISSUE THREAD

**Issue by emelyanovtv** [edenhill/librdkafka#3589]
Description
===========
It's probably a question: how to migrate from eager to cooperative rebalance easily (consumers)? 

According documentation migration is require 2 rolling bounces for consumer 
To upgrade from the old (eager) protocol, which always revokes all partitions before rebalancing, to cooperative rebalancing, you must follow a specific upgrade path to get all clients on the same ConsumerPartitionAssignor that supports the cooperative protocol. This can be done with two rolling bounces, using the CooperativeStickyAssignor for the example: during the first one, add "cooperative-sticky" to the list of supported assignors for each member (without removing the previous assignor -- note that if previously using the default, you must include that explicitly as well). You then bounce and/or upgrade it. Once the entire group is on 2.4+ and all members have the "cooperative-sticky" among their supported assignors, remove the other assignor(s) and perform a second rolling bounce so that by the end all members support only the cooperative protocol.

How to reproduce
================
Once I'm trying to add "cooperative-sticky" to the list of partition.assignment.strategy  I always see the same error message 

Checklist
=========
Please provide the following information:

 - [x] librdkafka version 1.7.0
 - [x] Apache Kafka version: `<REPLACE with e.g., 0.10.2.3>`
 - [x] librdkafka client configuration: `'partition.assignment.strategy': 'range,cooperative-sticky'`
 - [x] Operating system: `Alpine Linux v3.11 (x86_64 Linux)`
 - [x] Provide logs

Logs  (error message)
```
[LibrdKafkaError: All partition.assignment.strategy (range,cooperative-sticky) assignors must have the same protocol type, online migration between assignors with different protocol types is not supported] {
  origin: 'unknown',
  message: 'all partition.assignment.strategy (range,cooperative-sticky) assignors must have the same protocol type, online migration between assignors with different protocol types is not supported',
  code: -1,
  errno: -1
}
```

**Comment by g-clef** [comment]
@emelyanovtv I see that you closed this...did you resolve the problem another way?

**Comment by CTCC1** [comment]
Ran into similar issue to this today. I believe the error message holds true, i.e. "online migration between assignors with different protocol types is not supported."
I believe the documentation cited on[ 2 rounds of rolling upgrade for online migration between cooperative and non-cooperative protocol](https://cwiki.apache.org/confluence/display/KAFKA/KIP-429%3A+Kafka+Consumer+Incremental+Rebalance+Protocol#KIP429:KafkaConsumerIncrementalRebalanceProtocol-CompatibilityandUpgradePath) is specific to the Java based Clients.

I ended up doing an offline upgrade:
1) closing all consumers
2) change to use `"partition.assignment.strategy": "cooperative-sticky"`
